### DISCUSSION TITLE
Failed to load the extension Superbooga

### DISCUSSION THREAD

**Discussion by NekoTiga** [oobabooga/text-generation-webui#4902] [Q&A]
After finally getting all the needed dependencies thanks to https://github.com/oobabooga/text-generation-webui/issues/2279#issuecomment-1557563531

I keep getting this error
![image](https://github.com/oobabooga/text-generation-webui/assets/20857024/2d1db887-77d9-4f56-a15e-64748d323ad1)

Given my monkey brain no longer being able to process anything at this point, any chance anybody had a similar problem after chromadb and sentence_transformers installation for Superbooga?

Sorry in advance for stupid question, but I am a complete amateur.

**Comment by tmb80c** [comment]
Some issue on linux.

I used the cmd_linux.sh to enter into the conda environment.

Then executed as suggested:

pip install -r extensions/superbooga/requirements.txt --upgrade

Restarted and received a similar error like above.

Tried pip install chromadb

Doesn't solve the issue.
